contrib/resources/drives/y.conf

```diff
@@ -0,0 +1,21 @@
+# A directory holding 3rd party DOS tools that which can be manually
+# mounted or automounted using the "[dosbox] automount = true" setting.
+
+[drive]
+
+# Type of drive, where the possible values are
+# per the "mount -t" argument: dir, floppy, cdrom, or overlay.
+type = dir
+
+# Label of the drive, where the values are per the "mount -label" argument.
+label = addons
+
+# Path settings to apply, with the value formatted per the PATH command.
+path = %path%;y:\dos
```

Comment:
Yep, I like this approach better. This way we won't just take the drive letter `R` from the middle of all available letters 👍🏻